src/frontend/src/lib/types/dapp.ts

```diff
@@ -0,0 +1,26 @@
+// see https://github.com/dfinity/portal/tree/95c67a5cfe201e4e5cb79f3cf5d18fe16498cd8c?tab=readme-ov-file#object-schema
+export interface DApp {
```

Comment:
This will be replaced by some zod types once we implement the JSON file, correct? Maybe a TODO?

Reply:
Ah, no then let me add the zod schema directly